### PR TITLE
feat: Add Home and Explore Page

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
# Added Home and explore page 
1. Added UI for Home and Explore Page
2. Setup Backend for them
3. Filter by category
4. Individual Video Page feature

# Files to Review
1. RecommendationCard,jsx 
- I am using this component to show recommendations in individual video page

2. VideoCard,jsx 
- This component is used in Home and Explor page to show available videos

3. all-videos-context.js
- To make API call to backend and get all videos data

4. VideoPage.jsx
- This is individual page for each video

# Screenshots
## Home Page
Home Page

## Explore Page
Explore Page

## Individual Video Page
Individual Video Page

### CODE REVIEW

File: src/Pages/VideoPage/VideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    </div>
+                </div>
+
+                <hr></hr>
+
+                <h4>Comments</h4>
+            </div>
+            <div className='video-recommendation-container'>
+                {
+                    JSON.stringify(trendingVideosList)!==JSON.stringify([]) &&
```
Comment (anasm11) [inline src/Pages/VideoPage/VideoPage.jsx:113]: You can also do ` trendingVideosList.length && ` instead of this.

**Review by anasm11 — APPROVED** [review]
Code structure and ui looks good.

File: src/Components/Tabs/Tabs.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    padding: 1rem 2rem;
+    border-radius: 3rem;
+    font-family: Comfortaa;
+}
+
+/* .active-tab::before
+{
+    color: white;
+    background-color: black;
+}  */
```
Comment (nikhiljugale007) [inline src/Components/Tabs/Tabs.css]: You can remove commented code.

File: src/Components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <button 
+                onClick={()=>toggleTabHandler("superhero")} 
+                className={toggleTab==="superhero" ? "tab-buttons active-tab" : "tab-buttons"}>
+                    Super Hero
+            </button>
+            <button 
+                onClick={()=>toggleTabHandler("medievalFantasy")} 
+                className={toggleTab==="medievalFantasy" ? "tab-buttons active-tab" : "tab-buttons"}>
+                    Medieval Fantasy
+            </button>
```
Comment (nikhiljugale007) [inline src/Components/Tabs/Tabs.jsx:45]: suggestion: you can create separate component for individual tab and iterate over them on a map.

**Review by nikhiljugale007 — COMMENTED** [review]
Did not find any major issue. 
Very clean code with good naming convention. 
Good to merge.

File: src/Components/Tabs/Tabs.css (hunk 2 above)
Comment (Naman-Saxena1) [reply src/Components/Tabs/Tabs.css]: Done

**Pull request merged**